lucene/core/src/java/org/apache/lucene/search/MaxScoreAccumulator.java

```diff
@@ -35,9 +35,22 @@
     this.modInterval = DEFAULT_INTERVAL;
   }
 
-  void accumulate(int docID, float score) {
-    assert docID >= 0 && score >= 0;
-    long encode = (((long) Float.floatToIntBits(score)) << 32) | docID;
+  private static long maxBinary(long v1, long v2) {
```

Comment:
we call this `long` `encode` elsewhere, so maybe use the same convention for this function name?
```suggestion
  /**
   * Return the max encoded DocAndScore in a way that is consistent with {@link DocAndScore#compareTo}.
   */
  private static long maxEncode(long v1, long v2) {
```